packages/labs/virtualizer/package.json.bak

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
what's this file about?

Reply:
🤦  looks like I forgot I was making a backup file in my sed command in the migrate script that added internal-scripts...
```
sed -i.bak '85i\
    "internal-scripts": "^1.0.0",\
'  packages/labs/virtualizer/package.json
```

Fixing in migrate script.  Will remove that on next force push to branch.